test_args.sh

```diff
@@ -129,14 +129,14 @@ else
     kill $pid
 
     # Check that price can be set to 0 with -pricePerUnit 0
-    $TMPDIR/livepeer -orchestrator -serviceAddr 127.0.0.1:8935 -transcoder -s3bucket abc -pricePerUnit 0 -network rinkeby $ETH_ARGS 2>&1 | grep "Should specify both s3bucket and s3creds" 
+    $TMPDIR/livepeer -orchestrator -serviceAddr 127.0.0.1:8935 -transcoder -pricePerUnit 0 -network rinkeby $ETH_ARGS 2>&1 | grep "Should specify both s3bucket and s3creds" 
```

Comment:
This test needs to be fixed since the "Should specify both s3bucket and s3creds" message is no longer logged.

Reply:
fixed in 988a15a920cbcbbb5df64301b259b41547cd6112